tasks/deploy/config/sepolia/morpho-blue-dai-14day.ts

```diff
@@ -0,0 +1,140 @@
+            hre.hyperdriveDeploy.deployments.byName(
+                SEPOLIA_MORPHO_BLUE_COORDINATOR_NAME,
+            ).address,
+        deploymentId: toBytes32(SEPOLIA_MORPHO_BLUE_DAI_14DAY_NAME),
+        salt: toBytes32("0x420"),
+        extraData: enc,
+        contribution: CONTRIBUTION,
+        fixedAPR: parseEther("0.10"),
+        timestretchAPR: parseEther("0.10"),
+        options: async (hre) => ({
+            extraData: "0x",
+            asBase: true,
+            destination: (await hre.getNamedAccounts())["deployer"] as Address,
+        }),
+        prepare: async (hre, options) => {
+            let pc = await hre.viem.getPublicClient();
+            let baseToken = await hre.hyperdriveDeploy.ensureDeployed(
+                "DAI",
+                "ERC20Mintable",
+                [
+                    "DAI",
+                    "DAI",
+                    18,
+                    (await hre.getNamedAccounts())["deployer"] as Address,
+                    true,
+                    parseEther("10000"),
+                ],
+                options,
+            );
+
+            // approve the coordinator for the contribution
+            let tx = await baseToken.write.approve([
+                hre.hyperdriveDeploy.deployments.byName(
+                    SEPOLIA_MORPHO_BLUE_COORDINATOR_NAME,
+                ).address,
+                CONTRIBUTION,
+            ]);
+            await pc.waitForTransactionReceipt({ hash: tx });
+
+            // mint some tokens for the contribution
+            tx = await baseToken.write.mint([CONTRIBUTION]);
+            await pc.waitForTransactionReceipt({ hash: tx });
+        },
+        poolDeployConfig: async (hre) => {
+            return {
+                baseToken:
+                    hre.hyperdriveDeploy.deployments.byName("DAI").address,
+                vaultSharesToken: zeroAddress,
+                circuitBreakerDelta: parseEther("0.6"),
+                minimumShareReserves: parseEther("10"),
+                minimumTransactionAmount: parseEther("0.001"),
+                positionDuration: parseDuration("14 days"),
+                checkpointDuration: parseDuration("1 day"),
+                timeStretch: 0n,
+                governance: SEPOLIA_FACTORY_GOVERNANCE_ADDRESS,
+                feeCollector: SEPOLIA_FACTORY_GOVERNANCE_ADDRESS,
+                sweepCollector: SEPOLIA_FACTORY_GOVERNANCE_ADDRESS,
+                checkpointRewarder: hre.hyperdriveDeploy.deployments.byName(
+                    SEPOLIA_CHECKPOINT_REWARDER_NAME,
+                ).address,
+                ...(await getLinkerDetails(
+                    hre,
+                    hre.hyperdriveDeploy.deployments.byName(
+                        SEPOLIA_FACTORY_NAME,
+                    ).address,
+                )),
+                fees: {
+                    curve: parseEther("0.01"),
+                    flat: normalizeFee(parseEther("0.0005"), "14 days"),
```

Comment:
Might be good to make a constant for the position duration since we use `"14 days"` twice.